wolfcrypt/src/asn.c

```diff
@@ -2903,7 +2914,6 @@ int wc_CreatePKCS8Key(byte* out, word32* outSz, byte* key, word32 keySz,
         word32 tmpSz  = 0;
         word32 sz;
 
-
         /* If out is NULL then return the max size needed
```

Comment:
I added a sanity check, although seems odd. Calling `wc_CreatePKCS8Key` that already has a PKCS8 header...

Reply:
Thanks for adding check. I though that the use case of duplicate headers could happen when calling PEM_write_bio_PKCS8PrivateKey() with key including PKCS8 header instead of PEM_write_bio_PrivateKey(). But, it might be worried a lot.